server.js

```diff
@@ -27,6 +29,56 @@ app.get('/', function (req, res) {
      res.sendFile(__dirname + '/src/index.html');
 });
 
+//Marketing endpoint
+app.get('/marketing', function(req,res){
+    const con = mysql.createConnection(databaseConnectionOptions);
+    var today= new Date();
+    //Starting date of this week
+    var thisWeek = new Date();
+    thisWeek.setDate(today.getDate()-today.getDay());
+    //date from 2 weeks ago
+    var twoWeeks = new Date();
+    twoWeeks.setDate(today.getDate()-14);
+    //this month start date
+    var thisMonth = new Date();
+    thisMonth.setDate(1);
+    //last month start date
+    var lastMonth = new Date();
+    lastMonth.setDate(1);
+    lastMonth.setMonth(today.getMonth()-1);
+    //Get all registrations
+    var query = "SELECT COUNT(*) as total FROM str_register; ";
+    query+= "SELECT COUNT(*) as total FROM str_register WHERE created_at >= '" +thisWeek.toISOString().slice(0,10) +"';";
```

Comment:
ISO string is not as accurate as we need. Use moment.js to format the date in the format you need https://momentjs.com/

Reply:
This can be rewritten in modern javascript ES7 way ```SELECT COUNT(*) as total FROM str_register WHERE created_at >= '${thisWeek.toISOString().slice(0,10)}';``` with backquotes surrounding the code